client/modules/workspace/src/AppsSideNav/AppsSideNav.tsx

```diff
@@ -127,8 +122,8 @@ export const AppsSideNav: React.FC<{ categories: TAppCategory[] }> = ({
   const defaultKey = `${appId}${appVersion || ''}${currentApp?.bundle_id}`;
 
   return (
-    <div className={styles.appsBrowserSidebar}>
-      <h3>Applications:</h3>
+    <>
+      <h5 style={{ paddingLeft: 50 }}>Applications:</h5>
```

Comment:
This works, but since it seems the intent is to center the "Applications" header, are we able to use centering CSS like `justify-items: center`?